SPECS/gfs2-utils/gfs2-utils.spec

```diff
@@ -33,6 +30,7 @@ BuildRequires:  libtool
 BuildRequires:  libuuid-devel
 BuildRequires:  ncurses-devel
 BuildRequires:  zlib-devel
+BuildRequires:  bzip2-devel
```

Comment:
nit: I think BuildRequires packages here are sorted alphabetically, so could move this one up

Reply:
Thanks! Fixed it.